### DISCUSSION TITLE
AWS CLI GitHub Community Monthly Announcement May 2022

### DISCUSSION THREAD

**Discussion by kdaily** [aws/aws-cli#6958] [Announcements]
AWS CLI Community,

This is the May 2022 GitHub community monthly update post! We’ll use this format to provide updates related to community contributions, including highlights of contributions, updates to the process, and general team news.

## Overview

This month we commenced implementing and executing the new [community contributions process](https://github.com/aws/aws-cli/pull/6828). We have a team member each month who will be dedicated to community contributions on GitHub, so look for comments and updates from them on your contributions. In addition, we have set up a [GitHub Project board](https://github.com/orgs/aws/projects/21/views/2) that tracks GitHub issues from the contribution-ready stage through completion by merging a pull request. You can always access the GitHub Project board from the [“Projects” tab](https://github.com/aws/aws-cli/projects?type=beta) on the [AWS CLI GitHub repository](https://github.com/aws/aws-cli/). If you’re interested in finding an issue to contribute, the “Contribution Ready” lane on the project is the place to start!

We’ve also been working through the backlog of existing pull requests and determining which stage of the contribution process they are in. We started with 187 open pull requests, and in the last month have taken action on 51 issues. 18 of these issues have made it on to the project board, and 4 have been merged in.

We’ll continue spending most of our efforts working through the backlog of pull requests. We aren’t planning to move existing pull requests into the final review stage until we complete reviewing the backlog. In parallel, we’re working on writing the new contributing guide, using the proposal as a base. We’re tracking the work we’re doing related to the process on [this issue](https://github.com/aws/aws-cli/issues/6899).

## Feedback

Let us know through comments on this post of any feedback, ideas, or suggestions you have on the contribution process. Thanks!

**Comment by RedStarDreamer** [comment]
I am brand new to GitHub - I just signed up today. It looks like a vibrant community! I am currently working on the Google Data Analytics Certificate program.

**Comment by lorengordon** [comment]
How do I browse to the contributions proposal in the GitHub UI, starting from the default develop branch?

**Reply by kdaily** [reply]
Hi @lorengordon,

It's on the [v2 branch](https://github.com/aws/aws-cli/tree/v2), in the `proposals` directory. Here's a direct link:

https://github.com/aws/aws-cli/blob/v2/proposals/contribution-guide.md

**Comment by eightnoneone** [comment]
@kdaily Is this channel for monthly updates now abandoned? Is there any replacement? It appears to be the only post tagged as an [Announcement](https://github.com/aws/aws-cli/discussions/categories/announcements).
